python/ray/experimental/state.py

```diff
@@ -587,14 +589,16 @@ def micros_rel(ts):
             parent_info = task_info.get(
                             task_table[task_id]["TaskSpec"]["ParentTaskID"])
             worker = workers[info["worker_id"]]
-
+            # The catapult trace format documentation can be found here:
+            # https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH
```

Comment:
In this case it's ok to have a long line. I would do

```python
# https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview  # noqa: E501
```

the `noqa` thing prevents flake8 from complaining